modules/ROOT/pages/managing-users.adoc

```diff
@@ -14,12 +14,12 @@ Configuring SAML for SSO involves:
 == Prerequisites
 
 * Your Anypoint Platform organization must be set up as your audience.
-* The assertion consumer service must be set to send a `POST` request to `+https://anypoint.mulesoft.com/accounts/login/receive-id+`.
+* The assertion consumer service must be set to send a `POST` request to `+https://anypoint.mulesoft.com/accounts/login/:org-domain/providers/:providerId/receive-id+`.
```

Comment:
can we add a note that the providerId is available only after the provider configuration is created? since it's in the prerequisite section I wouldn't want it to cause confusion

Reply:
```suggestion
* The assertion consumer service must be set to send a `POST` request to `+https://anypoint.mulesoft.com/accounts/login/:org-domain/providers/:providerId/receive-id+`. +
Note that `:providerId` is available only after you create the provider configuration.
```